project.clj

```diff
@@ -4,18 +4,22 @@
   :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
             :url "https://www.eclipse.org/legal/epl-2.0/"}
   :dependencies [[org.clojure/clojure "1.10.1" :scope "provided"]
-                 [io.replikativ/datahike "0.3.2-SNAPSHOT" :exclusions [io.replikativ/konserve]]
+                 [io.replikativ/konserve "0.6.0-SNAPSHOT"]
+                 [io.replikativ/hitchhiker-tree "0.1.7"]
+                 [io.replikativ/datahike "0.3.2-SNAPSHOT" :exclusions [io.replikativ/konserve io.replikativ/hitchhiker-tree]]
```

Comment:
These dependencies will be removed once Datahike has a version that includes the correct dependency versions.

Reply:
Right, they should be removed. currently it does not work without them.